.github/workflows/chromatic-push.yml

```diff
@@ -0,0 +1,24 @@
+name: Chromatic - Push
+
+on: push
+
+jobs:
+  chromatic_review:
```

Comment:
What happens when a push is done to a branch that's not part of a PR? Is the review able to be processed?